client/src/components/ThreadE.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```

Comment:
What is this file for? How is it different than Thread.js?

Reply:
It's junk, you can delete it. Forgot to.